rootfs/api/tests/test_hooks.py

```diff
@@ -166,10 +166,10 @@ def test_build_hook_dockerfile(self):
         build = {'username': 'autotest', 'app': app_id}
         url = '/v2/hooks/builds'.format(**locals())
         SHA = 'ecdff91c57a0b9ab82e89634df87e293d259a3aa'
-        DOCKERFILE = """
-        FROM busybox
-        CMD /bin/true
-        """
+        DOCKERFILE = """\
+FROM busybox
+CMD /bin/true"""
```

Comment:
That'd be the same as:

```
        DOCKERFILE = """\
FROM busybox
        CMD /bin/true"""
```

But that one does look weird, hah. I changed the PR for now  and kept that spacing there as it doesn't hurt.